docs/ENV.md

```diff
@@ -54,6 +54,19 @@ These variables are specifically for a single language or tool.
+| COCOA_EXCLUDED_TARGETS                  | Comma-separated list of targets to exclude from resolution of dependencies. If a target has nested targets, these are also excluded. For excluding only those nested targets, use a `/` to describe the target, eg `target/subtarget/subsubtarget` -- this excludes only `subsubtarget` from the resolution.                                                                                                                                                                                                                         |
+| COCOA_FULL_SCAN                         | Whether or not to do a full (deep) scan of the pods. This requires CocoaPods to be installed and runnable from the `PATH`. When set to `false` or `0`, only the most basic of information will be gathered (name, version, purl and, if applicable, sub-spec) -- can be useful if run on Windows. Defaults to `true`.                                                                                                                                                                                                                |
+| COCOA_INCLUDED_TARGETS                  | Comma-separated list of target to include for resolution of dependencies. Usage is the same as `COCOA_EXCLUDED_TARGETS` above.                                                                                                                                                                                                                                                                                                                                                                                                       |
+| COCOA_MERGE_SUBSPECS                    | Should all sub-specs (and their dependencies) be merged into the root spec. This can be useful if you are importing your SBOM into a tool that doesn't handle sub-specs as separate dependencies (eg dependency-track). Defaults to `false`.                                                                                                                                                                                                                                                                                         |
+| COCOA_PODSPEC_JSON_REPLACEMENTS         | Comma-separated list of 'text_to_find=replacement'. This can be used if you have local pods in JSON format that are not (correctly) parsable by the `pod ipc sepc`-command, eg because some variables or functions would be inserted by the `Podfile` -- users of react-native / expo might need this. This will be interpreted as a `regex` if the 'text_to_find' is both prefixed and suffixed with a `/` and then references like `$1` can be used in the 'replacement'. To find or insert a newline, use the String `<NEWLINE>`. |
```

Comment:
Type `sepc` to `spec`

Reply:
This is introducing some code injection risks. Can you share any documentation about this command?

Reply:
Shall I add a completely new page, or add it here? I feel this description is already pretty much at the limit of what is practical for a quick overview of the EnvVars...

Reply:
Maybe a console.log? If not don't worry, we can work on a new PR.

Reply:
OK, let's do this in a new PR then.